packages/web/src/utils/token-utils.ts

```diff
@@ -212,6 +212,11 @@ export const formatTokenModelPath = (token: TokenModel): string => {
   return token.path.replace(/^gno\.land\//, "");
 };
 
+export const isNativeTokenPath = (token: TokenModel): boolean => {
+  if (token.path === GNOT_TOKEN.path) return true;
+  return false;
```

Comment:
we can make this more simpler way.
```suggestion
  return token.path === GNOT_TOKEN.path;
```